toolkit/tools/internal/sliceutils/sliceutils.go

```diff
@@ -55,3 +55,21 @@ func StringsSetToSlice(inputSet map[string]bool) []string {
 
 	return outputSlice[:index]
 }
+
+// RemoveDuplicateStrings will remove duplicate entries from a string slice
+func RemoveDuplicateStrings(packList []string) (deduplicatedPackList []string) {
+	var (
+		packListSet = make(map[string]struct{})
+		exists      = struct{}{}
+	)
+
+	for _, entry := range packList {
+		packListSet[entry] = exists
+	}
+
+	for entry := range packListSet {
+		deduplicatedPackList = append(deduplicatedPackList, entry)
+	}
+
+	return
```

Comment:
An attempt to simplify:
```suggestion
	packListSet := map[string]bool{}

	for _, entry := range packList {
		packListSet[entry] = true
	}

	return StringsSetToSlice(packListSet)
```

Reply:
Tested and confirmed this works :)